aws_topology/stackstate_checks/aws_topology/resources/route53_hostedzone.py

```diff
@@ -1,40 +1,103 @@
-from .utils import make_valid_data
+from .utils import (
+    make_valid_data,
+    create_arn as arn,
+    client_array_operation,
+    set_required_access_v2,
+    transformation,
+)
 from .registry import RegisteredResourceCollector
+from collections import namedtuple
+from schematics import Model
+from schematics.types import StringType, ModelType, ListType
+
+
+def create_arn(region=None, account_id=None, resource_id=None, **kwargs):
+    return arn(resource="route53", region="", account_id="", resource_id="hostedzone/" + resource_id)
+
+
+class HostedZoneInfo(Model):
+    Id = StringType(required=True)
+    Name = StringType(default="UNKNOWN")
```

Comment:
Do we really need this default? If we can't get the name can we uniquely identify this hosted zone?

Reply:
I see we use the hosted zone id as the external id, maybe we can fallback to that instead of unknown for the name

Reply:
You will probably have to keep this here then and add a check in `process_hosted_zone`